README.md

```diff
@@ -1,149 +1,114 @@
-  5. Since this is a mobile web application, webapp should be viewed in a mobile view in the browser
-
-# Testing:
-
-### Webapp:
-  Automated testing for the front-end is done using [cypress](https://www.cypress.io/).
-  To run automated tests in a chrome browser:
-  - Ensure that the webapp portal is running by doing `npm start` in the root directory of the webapp package
-  - `npm run cypress-ui` in the root directory of the webapp package
-
-  The test modules are stored in `packages/webapp/cypress/integration`.
-  Additional tests can be added by adding more testing modules to this directory.
-
-  Unit tests can also be run by:
-  - `npm run test`.
-
-  Unit tests reside in `src/tests`, and additional unit tests can be added in this directory.
-
-  We are currently using [ESLint](https://eslint.org/) to maintain code style in the app.
-  The linter can be run by `npm run lint`.
-
-### Api:
-  There is currently end-to-end testing being done for the LiteFarm API using the [chai.js](https://www.chaijs.com/)
-  and [jest](https://jestjs.io/) libraries.
-  The test files can be found in `packages/api/tests`.
-
-  All of the tests run real queries to the database using a jwt token obtained through Auth0.
-  Therefore, locally run tests can affect the state of the db. This is addressed by running a script which deletes
-  test data that is generated during the tests. However, this is not fool-proof, and
-  it may be necessary to clear the local db in some cases.
-
-  To run an end-to-end test, `npm run e2e` in the api directory.
-
-  To run a test file with a specific name:
-   - `npm start`
-   - `jest test_name`
-
-  It should be noted that any merges to develop and master branches in github will result in the code going through a CI pipeline, which
-  runs the end-to-end test. Therefore it is important to test locally first before creating any pull requests
-  to develop.
-
-  We are currently using [ESLint](https://eslint.org/) to maintain code style in the app. The linter can be run by `npm run lint`.
+- `packages/api`: the back-end API server
+- `packages/webapp`: the client-facing application
+
+## Preliminaries 
+
+1. Install [node.js](https://nodejs.org/en/download/package-manager/) if you do not already have it.
+3. Use the `git clone` command to clone this repository to your computer.
+
+## Configure LiteFarm applications
+
+The applications are configured with environment variables stored in `.env` files. Configuration information includes secrets like API keys, so the `.env` files should never be added to source control. This repository does contain `.env.default` files for api and webapp. Begin with these, and customize as needed.
+
+1. Create file `packages/api/.env` by copying the `.env.default` file in that folder. 
+
+2. Create file `packages/webapp/.env` by copying the `.env.default` file in that folder. For webapp to work, you must edit your new `.env` file to provide values for two required environment variables:
```

Comment:
"Create file `packages/webapp/.env` by copying the `.env.default` file in that folder." has been said twice

Reply:
It needs to happen twice-- once in each package. But I'll try to make it clearer.

Reply:
oh oops, I misread. sorry about that